src/main/rx/crsf.c

```diff
@@ -130,7 +136,7 @@ typedef struct crsfPayloadRcChannelsPacked_s crsfPayloadRcChannelsPacked_t;
  * int8_t Uplink SNR ( db )
  * uint8_t Diversity active antenna ( enum ant. 1 = 0, ant. 2 )
  * uint8_t RF Mode ( enum 4fps = 0 , 50fps, 150hz)
- * uint8_t Uplink TX Power ( enum 0mW = 0, 10mW, 25 mW, 100 mW, 500 mW, 1000 mW, 2000mW )
+ * uint8_t Uplink TX Power ( enum 0mW = 0, 10mW, 25 mW, 100 mW, 500 mW, 1000 mW, 2000mW, 250mW )
```

Comment:
It is possible to put 250mw after 100mw here?

Reply:
@haslinghuis No, because the Rx sends an enum item, not a value. Since 250mW is added aferwards, its value is always 7.

Reply:
I think that comment should follow consistency of the enum.

Reply:
@haslinghuis , because of this: `enum 0mW = 0,..` in a full description this would be `* uint8_t Uplink TX Power ( enum 0mW = 0, 10mW = 1, 25 mW = 2, 100 mW = 3, 500 mW = 4, 1000 mW = 5, 2000mW = 6, 250mW = 7 )`. If we write this description line like `* uint8_t Uplink TX Power ( enum 0mW = 0, 10mW = 1, 25 mW = 2, 100 mW = 3, 250mW = 4, 500 mW = 5, 1000 mW = 6, 2000mW = 7 )` we'll mess up a description, because it's totally wrong after 100mW item.

Reply:
If TBS decides the order nothing we can do about it without a lot of hassle when they add another entry 😛

Reply:
@haslinghuis , but on this way, we exactly know what is added last. So, if TBS decide to add more power levels, it will be 8, then 9, etc., in the enum. But.. if they decide to reorder whole enum, then.. we will have slightly bigger problem than a just one line of comment code. 😛